File: benchmark/c/cpw/minimize_the_sum_of_digits_of_a_and_b_such_that_a_b_n/minimize_the_sum_of_digits_of_a_and_b_such_that_a_b_n.c


#include <stdio.h>
#include <math.h>
#include <stdlib.h>
#include <limits.h>
#include <stdbool.h>



int minimize_the_sum_of_digits_of_a_and_b_such_that_a_b_n ( int n ) ;
int minimize_the_sum_of_digits_of_a_and_b_such_that_a_b_n ( int n ) {
  int sum = 0;
  while ( n > 0 ) {
    sum += ( n % 10 );
    n /= 10;
  }
  if ( sum == 1 ) return 10;
  return sum;
}

File: translation_result/else_bms/success/c/cpw/minimize_the_sum_of_digits_of_a_and_b_such_that_a_b_n/mistral/BaseRepair/minimize_the_sum_of_digits_of_a_and_b_such_that_a_b_n.rs


fn minimize_thesum_of_digits_of_a_and_b_such_that_a_b_n(n: i32) -> i32 {
    let mut sum = 0;
    let mut n = n;

    while n > 0 {
        sum += n % 10;
        n /= 10;
    }

    if sum == 1 {
        10
    } else {
        sum
    }
}
